moto/redshiftdata/models.py

```diff
@@ -0,0 +1,192 @@
+
+            if statement.Status != "STARTED":
+                raise ValidationException(
+                    "Could not cancel a query that is already in %s state with ID: %s"
+                    % (statement.Status, statement_id)
+                )
+
+            statement.Status = "ABORTED"
+            self.statements[statement_id] = statement
+        except KeyError:
+            # Statement does not exist.
+            raise ResourceNotFoundException()
+
+        return True
+
+    def describe_statement(self, statement_id):
+        _validate_uuid(statement_id)
+
+        try:
+            # Statement exists
+            return self.statements[statement_id]
+        except KeyError:
+            # Statement does not exist.
+            raise ResourceNotFoundException()
+
+    def execute_statement(
+        self,
+        cluster_identifier,
+        database,
+        db_user,
+        parameters,
+        secret_arn,
+        sql,
+    ):
+        statement = Statement(
+            cluster_identifier=cluster_identifier,
+            database=database,
+            db_user=db_user,
+            query_parameters=parameters,
+            query_string=sql,
+            secret_arn=secret_arn
+        )
+        self.statements[statement.Id] = statement
+        return statement
+
+    def get_statement_result(self, statement_id):
+        _validate_uuid(statement_id)
+
+        if statement_id not in self.statements:
+            raise ResourceNotFoundException()
+
+        # Return static statement result
+        # StatementResult cannot be mocked because it's the result of the SQL query passed as parameter
+        return StatementResult(
+            [
+                vars(ColumnMetadata(None, False, True, "Number", False)),
+                vars(ColumnMetadata(None, True, False, "Street", False)),
+                vars(ColumnMetadata(None, True, False, "City", False)),
+            ],
+            [
+                [
+                    vars(Record(long_value=10)),
+                    vars(Record(string_value="Alpha st")),
+                    vars(Record(string_value="Vancouver"))
+                ],
+                [
+                    vars(Record(long_value=50)),
+                    vars(Record(string_value="Beta st")),
+                    vars(Record(string_value="Toronto"))
+                ],
+                [
+                    vars(Record(long_value=100)),
+                    vars(Record(string_value="Gamma av")),
+                    vars(Record(string_value="Seattle"))
+                ]
+            ],
+            3,
+        )
+
+
+def _validate_uuid(uuid):
+    match = re.search(r'^[a-z0-9]{8}(-[a-z0-9]{4}){3}-[a-z0-9]{12}(:\d+)?$', uuid)
+    if not match:
+        raise ValidationException("id must satisfy regex pattern: ^[a-z0-9]{8}(-[a-z0-9]{4}){3}-[a-z0-9]{12}(:\\d+)?$")
+
+
+# For unknown reasons I cannot use the service name "redshift-data" as I should
```

Comment:
Yes - that is a known issue, there are several services that have the same problem. Using the `redshift`-service is a good workaround until boto3 fixes this